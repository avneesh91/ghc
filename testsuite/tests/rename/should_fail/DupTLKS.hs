{-# LANGUAGE TopLevelKindSignatures #-}

module DupTLKS where

import Data.Kind (Type)

data D

type D :: Type
type D :: Type
type D :: Type
