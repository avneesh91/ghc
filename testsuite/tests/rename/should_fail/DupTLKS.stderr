
DupTLKS.hs:9:1: error:
    Duplicate top-level kind signatures for ‘D’:
      type D :: Type -- written at DupTLKS.hs:9:1-9
      type D :: Type -- written at DupTLKS.hs:10:1-9
      type D :: Type -- written at DupTLKS.hs:11:1-9
